cypress/TestCases/FireboltCertification/UserGrants.feature

```diff
@@ -0,0 +1,788 @@
+    # Case-1: Deny grant and validate error
+    # Case-2: Reboot device after ttl, after reboot when api is called again, expect to ask grant and then deny grant
+    @Usergrants @coreSDK @sdk @transport @notSupported @requiresPlatformImplementation
+    Scenario: UserGrants.Capabilities R*2.2.2.5 - Negative Scenario: Validate Capabilities with denied (ackchallenge) and lifespan:seconds lifespanTtl:60sec - reboot after ttl and it should ask grant
+        Given the environment has been set up for 'userGrants' tests
+        And 3rd party 'certification' app is launched
+        # Case-1
+        And Framework registers 'ackchallenge' test provider
+        And User 'starts' recording lifecycle history for '3rd party app'
+        When '3rd party app' invokes the 'Firebolt' API to 'expect error for device id'
+        Then 'Firebolt' platform responds with 'invalid request for device id'
+        When '3rd party app' invokes the 'Firebolt' API to 'check if capabilities is granted for device id'
+        Then 'Firebolt' platform responds with 'false for capabilities granted'
+        When User 'stops' recording lifecycle history for '3rd party app'
+        Then User validates lifecycle history for '3rd party app' with 'background:foreground'
+        # Case-2
+        # rebooting the device after ttl and validating again
+        ### When I reboot device
+        When 3rd party 'certification' app is launched
+        And User set response for 'set acknowledge granted'
+        And Framework registers 'ackchallenge' test provider
+        And User 'starts' recording lifecycle history for '3rd party app'
+        And '3rd party app' invokes the 'Firebolt' API to 'fetch device id'
+        Then 'Firebolt' platform responds with 'expected device id'
+        When '3rd party app' invokes the 'Firebolt' API to 'check if capabilities is granted for device id'
+        Then 'Firebolt' platform responds with 'true for capabilities granted'
+        When User 'stops' recording lifecycle history for '3rd party app'
+        Then User validates lifecycle history for '3rd party app' with 'background:foreground'
+
+    # Testing with grantPolicy having not supported capability, so it will return error since capability is not supported
+    @Usergrants @coreSDK @sdk @transport @requiresPlatformImplementation
+    Scenario: UserGrants.Capabilities - Negative Scenario: Validate Capability Without supported capability expecting error
+        Given the environment has been set up for 'userGrants' tests
+        And 3rd party 'certification' app is launched
+        And Framework registers 'pinChallenge' test provider
+        # use localization.laton capability which is not supported xrn:firebolt:capability:localization:location
+        When '3rd party app' invokes the 'Firebolt' API to 'expect error for localization latlon'
+        Then 'Firebolt' platform responds with 'not supported error for localization latlon'
+
```

Comment:
how do you know latlon is not supported for all platforms ? I believe this was mentioned as additional test case in the excel

Reply:
where is the readme document explaining about generic instructions on how to run usergrants >?
Also why is grantPolicy.json not added in PR? Can it be included in FCS PR or does it have to reside in configModule ? Confirm that and give proper explanation on where and why grantPolicy was added

Reply:
The Latlon test case has been moved to additional testcases, and the key has been generalized. A comment has also been added to the test case to use of not supported capability.

Reply:
Added usergrants readme document with basic instructions and added grantpolicy.json